content/v2.1/helpers/form_helper.md

```diff
@@ -0,0 +1,964 @@
+
+# With HTML Attributes
+f.time_field("book.release_hour", class: "form-control")
+# =>
+<input type="time" name="book[release_hour]" id="book-release-hour" value="" class="form-control">
+```
+### month_field
+Returns a month input tag.
+
+```ruby
+f.month_field("book.release_month")
+# =>
+<input type="month" name="book[release_month]" id="book-release-month" value="">
+
+# HTML Attributes
+f.month_field("book.release_month", class: "form-control")
+# =>
+<input type="month" name="book[release_month]" id="book-release-month" value="" class="form-control">
+```
+### week_field
+   Returns a week input tag.
+
+```ruby
+
+f.week_field("book.release_week")
+# =>
+<input type="week" name="book[release_week]" id="book-release-week" value="">
+
+# With HTML Attributes
+f.week_field("book.release_week", class: "form-control")
+# =>
+<input type="week" name="book[release_week]" id="book-release-week" value="" class="form-control">
+```
+### email_field
+Returns an email input tag.
+
+```ruby
+f.email_field("user.email")
+# =>
+<input type="email" name="user[email]" id="user-email" value="">
+
+# With HTML Attributes
+f.email_field("user.email", class: "form-control")
+# =>
+<input type="email" name="user[email]" id="user-email" value="" class="form-control">
+```
+### url_field
+
+Returns a URL input tag.
+
+```ruby
+f.url_field("user.website")
+# =>
+<input type="url" name="user[website]" id="user-website" value="">
+
+# With HTML Attributes
+f.url_field("user.website", class: "form-control")
+# =>
+<input type="url" name="user[website]" id="user-website" value="" class="form-control">
+```
+### tel_field
+Returns a telephone input tag.
+
+   ```ruby
+f.tel_field("user.telephone")
+# =>
+<input type="tel" name="user[telephone]" id="user-telephone" value="">
+
+# With HTML Attributes
+f.tel_field("user.telephone", class: "form-control")
+# =>
+<input type="tel" name="user[telephone]" id="user-telephone" value="" class="form-control">
+```
+
+### hidden_field
+Returns a hidden input tag, not visible in the rendered page but only in the source code.
+
+```ruby
+f.hidden_field("delivery.customer_id")
+# =>
+<input type="hidden" name="delivery[customer_id]" id="delivery-customer-id" value="">
+```
+
+There is hardly a need to pass HTML attributes to this one, but you can of course do this as with any other input field.
+### file_field
+Returns a file input tag. It allows you to upload a file and accepts two special attributes:
+
+```ruby
+f.file_field("user.avatar")
+# =>
+<input type="file" name="user[avatar]" id="user-avatar">
+
+# With  HTML Attributes
+f.file_field("user.avatar", class: "avatar-upload")
+# =>
+<input type="file" name="user[avatar]" id="user-avatar" class="avatar-upload">
+```
+
+**Specifying accepted MIME Types**
+
+You can control what MIME Types your input allow by passing in the additional option `:accepted`. This can be a `String`, where types are separated by coma, or an `Array`.
```

Comment:
```suggestion
You can control what MIME Types your input allow by passing in the additional option `:accepted`. This can be a `String`, where types are separated by comma, or an `Array`.
```